backend/src/models/service.ts

```diff
@@ -94,6 +94,20 @@ export class Service extends BaseEntity {
   })
   discoveredBy: Scan;
 
+  /** Scan that discovered this product/cpe (wappalyzer, intrigue-ident, censys). */
```

Comment:
```suggestion
  /** Name of scan that discovered this product/cpe (wappalyzer, intrigue-ident, censys). */
```

Reply:
What does "this product/cpe" mean? I believe a service has multiple products. Did you mean to say "the products/cpes on this service"?